src/saving/NewSaveMenu.cs

```diff
@@ -102,9 +107,16 @@ private void SaveButtonPressed()
 
     private void OnConfirmSaveName()
     {
-        GUICommon.Instance.PlayButtonPressSound();
```

Comment:
Shouldn't the button press sound still be played?

Reply:
This function is only called from one function above,  in which the button press sound is already played.

Reply:
That's fine, then.